static/bidder-info/ownadx.yaml

```diff
@@ -0,0 +1,12 @@
+endpoint: "http://{{.Host}}-prebid.owadx.com/bidder/bid/{{.AccountID}}/000?token={{.SourceId}}"
+maintainer:
+  email: prebid-team@techbravo.com
```

Comment:
@ownAdx-prebid  

prebid team has sent verification mail on  `prebid-team@techbravo.com` .  Please respond back with "received" message to complete verification

Reply:
received response. 👍🏼